python_frank_energie/models.py

```
@@ -264,82 +1468,933 @@ def today(self) -> list[Price]:
+        if not tomorrow_prices:
+            return None
+
+        average_price = mean(price.marketPrice for price in tomorrow_prices)
+        rounded_average_price = round(average_price, DEFAULT_ROUND)
+
+        return rounded_average_price
+
+    @property
+    def tomorrow_min(self) -> Optional[Price]:
+        """Price with the lowest total for today."""
+        if not self.tomorrow == []:
```

Ruff

Use `self.tomorrow != []` instead of `not self.tomorrow == []`

Replace with `!=` operator

(SIM201)